test/integration/error-node-builtin-unresolved/expected-output.win.txt

```diff
@@ -0,0 +1,6 @@
+[snowpack] ! installing dependencies…
+[snowpack] "XXXX" (Node.js built-in) could not be resolved. (https://www.snowpack.dev/#node-built-in-could-not-be-resolved). See https://www.snowpack.dev/#troubleshooting
+[snowpack] Entry module cannot be external (http).
+[snowpack] install skipped (nothing to install)
+[snowpack] ../../../node_modules/bad-node-builtin-pkg/entrypoint.js
+   "XXXX" (Node.js built-in) could not be resolved. (https://www.snowpack.dev/#node-built-in-could-not-be-resolved)
```

Comment:
@FredKSchott I tried

At least we sorta re-enabled this test! Now we’re not skipping anything 🎉

Reply:
this feels like a great solution